components/twist/sources/new-event/new-event.js

```diff
@@ -30,12 +45,16 @@ module.exports = {
       };
     },
     getMeta(body) {
-      const { name, id, created } = body;
-      return {
+      const {
+        name,
         id,
+        created = Date.now(),
+      } = body;
+      return {
+        id: `${id}${created}`,
         summary: name || "New Event",
         ts: Date.parse(created),
```

Comment:
If `created` is set with `Date.now()`, then this call will return a `NaN`:
```
> Date.parse(Date.now())
NaN
```

```suggestion
        id,
        name: summary = "New Event",
        created = new Date(),
      } = body;
      const ts = Date.parse(created);
      return {
        id: `${id}${ts}`,
        summary,
        ts,
```

Reply:
Updated